code/game/objects/items/storage/belt.dm

```diff
@@ -328,8 +328,12 @@
 
 
 /obj/item/storage/belt/shotgun/attackby(obj/item/I, mob/user, params)
-	if(istype(I, /obj/item/ammo_magazine/shotgun))
-		var/obj/item/ammo_magazine/shotgun/M = I
+	
+	if(istype(I, /obj/item/ammo_magazine/handful))
```

Comment:
is this the best way to have handfuls have regular functionality, but other mags to empty?